### PR TITLE
feat: added custom tailwind config  and values component updated

### PR DESCRIPTION
**Pull request by amanuela97**
## Core values 

- Create core values section of homepage
- Open dev Tools -> Ctrl + Shit + M  -> To view on smaller devices
- [demo](https://wehelpdemo.netlify.app/)

### CODE REVIEW

**Review by razbakov — CHANGES_REQUESTED** [review]
@amanuela97 please make sure visually it looks same as on Figma: fonts, position of text, etc.

File: nuxt.config.js [hunk 1]
```diff
@@ -40,6 +40,14 @@ export default defineNuxtConfig({
       },
     ],
   },
+  buildModules: ['@nuxtjs/google-fonts'],
+  googleFonts: {
+    display: 'swap',
+    families: {
+      'Nunito sans-serif': true,
```
Comment (razbakov) [inline nuxt.config.js:47]: Check generated css code and network tab. I guess problem is here

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by amanuela97** [comment]
> @amanuela97 please make sure visually it looks same as on Figma: fonts, position of text, etc.

File: components/block/values.vue [hunk 2]
```diff
@@ -1,78 +1,13 @@
...
-        <strong>hope</strong> and help them through effective
-        <strong>actions</strong>.
+  <div class="flex flex-col min-h-screen ">
+      <h2 class="ml-[25px] mr-[198px] mb-[45px]  w-[152px] h-[21px] font-nunito text-headline-6 font-semibold text-[#2b2b2b]">Our core values</h2>
+
+      <p class="ml-[28px] mr-[16px] mb-[30px] w-[331px] h-[66px] font-nunito text-subtitle font-normal text-[#464646]"> 
+        We aim to be a <span class="font-bold text-green-500">trustful</span> source that give
+        people <span class="font-bold text-orange-500">hope</span> and help them through <span class="font-bold text-blue-500">effective</span> 
+        actions
       </p>
```
Comment (harshbrahmxatriya) [inline components/block/values.vue:9]: Is prettier working when you save the file?

**Review by harshbrahmxatriya — DISMISSED** [review]

**Comment by razbakov** [comment]
@amanuela97 please resolve the conflict

**Review by razbakov — CHANGES_REQUESTED** [review]
@amanuela97 check demo on desktop. Does it look OK to you? Please use some of TailwindUI components, that are responsive. Make it look good on desktop.

**Comment by amanuela97** [comment]
> @amanuela97 check demo on desktop. Does it look OK to you? Please use some of TailwindUI components, that are responsive. Make it look good on desktop.

should position of text be same on dekstop as in mobile ?

**Review by razbakov — APPROVED** [review]
LGTM

**Pull request merged**